packages/shared/src/lib/links.ts

```diff
@@ -1,2 +1,15 @@
 export const getTagPageLink = (tag: string): string =>
   `${process.env.NEXT_PUBLIC_WEBAPP_URL}tags/${encodeURIComponent(tag)}`;
+
+const previewDeployments = [
+  'https://preview.app.daily.dev',
+  'https://preview2.app.daily.dev',
+];
+
+export const checkIsPreviewDeployment = (): boolean => {
```

Comment:
May I suggest using environment variables for this? We can set Vercel to embed an env var for every preview deployment. Much more flexible than checking the domains

Reply:
Surely. I will come up with a more descriptive variable name in the case.